dev-python/spyder-vim/spyder-vim-9999.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 )
+
+inherit distutils-r1 git-r3 virtualx
+
+DESCRIPTION="Plugin for Spyder to enable Vim keybindings"
+HOMEPAGE="https://github.com/spyder-ide/spyder-vim"
+EGIT_REPO_URI="https://github.com/spyder-ide/spyder-vim.git"
+
+IUSE="doc"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS=
+
+RDEPEND="app-editors/vim
+	<dev-python/spyder-4.0.0[${PYTHON_USEDEP}]"
+
+DEPEND="test? (
+	dev-python/mock[${PYTHON_USEDEP}]
+	dev-python/pytest-qt[${PYTHON_USEDEP}] )"
+
+distutils_enable_tests pytest
+
+python_test() {
+	virtx pytest -vv
+}
+
+python_install_all() {
+	use doc && dodoc -r doc/*
+
+	distutils-r1_python_install_all
+}
```

Comment:
Depending on size etc. I'd say it's worth to make into a USE flag once it becomes multiple megabytes large, like +5 MB, or has hundreds of small files.